### ISSUE TITLE
No traffic stats - Raspbian Buster (similar to #1010)

### ISSUE THREAD

**Issue by ph0real** [RaspAP/raspap-webgui#1085]
## Checklist

- [ x] This is a bug report
- [x ] I have searched this repository for existing issues
- [ ] I checked the FAQ and official documentation before creating this issue
- [ ] I have read and understand the issue reporting guidelines 

## Your environment
1. Operating System: RPI 32 bit Buster
2. Hardware and version: RPI 4
3. RaspAP version: 2.71
4. Followed the project prerequisites? yes
5. RaspAP Quick Install or Manual setup? quick
6. Using default configuration? yes
7. Simultaneous AP and managed mode? no
8. Onboard wireless chipset or external adapter? onboard
9. Other software or services running with RaspAP? no

## Steps to reproduce
Install RASPAP with the bash script on Raspbian Buster. 
Do not select VPN and/or AD filtering. Using DHCPCD as prescribed.
Hotspot is functioning fine after basic setup with the webGUI.
VNstat version = 1.18
When I run VNstat manually, it does show traffic stats.

## Screenshots
![vnstat](https://user-images.githubusercontent.com/19611775/146533587-9bd467fe-b847-4585-a788-6c90e0d20e56.png)
![raspap](https://user-images.githubusercontent.com/19611775/146533589-a4842149-df46-49ad-863e-fd087e727bd2.png)

**Comment by billz** [comment]
See the [release notes](https://github.com/RaspAP/raspap-webgui/releases/tag/2.7.0).

tl;dr: follow the project prerequisites and install Bullseye.
